messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/AmqStreamsOpenShiftIT.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.ts.openshift.messaging.kafka;
+
+import io.quarkus.ts.openshift.common.AdditionalResources;
+import io.quarkus.ts.openshift.common.OnlyIfConfigured;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+
+@OpenShiftTest
+@AdditionalResources("classpath:ocp-amq-streams.yaml")
```

Comment:
In the rest of the modules, the resource names do not mention the target cluster (which is always oc), just the component name to be installed.